src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/AssessmentStatus.kt

```diff
@@ -134,6 +143,11 @@ enum class AssessmentStatus {
   },
 
   APPROVED {
+    override fun tasks() = mapOf(
+      PRISON_DM to listOf(
+        TaskProgress.Fixed(APPROVE_LICENCE, READY_TO_START),
```

Comment:
mmmh.. Is this ready to start? not in_progress?

Reply:
Ya it says ready to start for APPROVED state.
![image](https://github.com/user-attachments/assets/ce174529-7eda-45ca-a14d-a6830be9da2d)